auth/server/base/src/main/java/org/wildfly/security/auth/server/SecurityRealm.java

```diff
@@ -64,6 +65,25 @@ default RealmIdentity getRealmIdentity(Evidence evidence) throws RealmUnavailabl
         return principal == null ? RealmIdentity.NON_EXISTENT : getRealmIdentity(principal);
     }
 
+    /**
+     * Get a handle for to the identity for the given evidence in the context of this security realm. Any validation / name
+     * mapping is an implementation detail for the realm. The identity may or may not exist. The principal obtained is
+     * transformed prior to obtaining the identity. The returned handle <em>must</em> be cleaned up by a call to
```

Comment:
"The principal obtained is transformed prior to obtaining the identity."

^ I think this sentence should be updated to clarify that that's for the authentication identity.

Reply:
Fixed